client/src/graphs/templates/templates.test.ts

```diff
@@ -0,0 +1,91 @@
+import { describe, it, expect } from "vitest";
+import {
+  getAverageCoordinatesOfGraphNodes,
+  centerNodesOnOriginCoordinates,
+} from "./helpers";
+
+describe("getAverageCoordinatesOfGraphNodes", () => {
+  it("should return the average coordinates for a list of nodes", () => {
```

Comment:
no! ❌  ~~it("should return the average coordinates for a list of nodes"~~ 

✅ it("returns the average coordinates of the nodes"